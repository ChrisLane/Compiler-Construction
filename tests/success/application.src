// This tests the Application expression. The code should parse.
main() {
   f(x);
}
