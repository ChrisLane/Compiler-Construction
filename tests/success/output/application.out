main (  ) { 
    Application ( Identifier "f" ( Identifier "x" ) ) 
}
